src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+            <v-col cols="auto">
+              <v-icon>mdi-information-outline</v-icon>
+            </v-col>
+            <v-col class="pl-2" style="max-width: 50rem">
+              <div v-if="item.info === ITEM_TYPES.SIMILAR_MATCH">
+                <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+                  <template v-slot:activator="{ on, attrs }">
+                    <span class="dotted-underline" v-bind="attrs" v-on="on">Similar names</span>
+                  </template>
+                  <p class="ma-0">
+                    Similar names contain the same, synonymous, or similar looking or sounding words.
+                  </p>
+                </v-tooltip>
+                in different business categories may be approved at the discretion of the
+                Business Registry. If you feel your name is unique within your business category,
+                or you can
+                <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+                  <template v-slot:activator="{ on, attrs }">
+                    <span class="dotted-underline" v-bind="attrs" v-on="on">obtain consent</span>
+                  </template>
+                  <p class="ma-0">
+                    You may be able to use a similar name if you obtain consent or authorization
+                    from the business using the name. If consent is required you will be asked
+                    to submit signed consent to the Business Registry as part of the review process.
+                  </p>
+                </v-tooltip>
+                to use the name, you can submit your name for review.
+              </div>
+              <div v-else-if="item.info === ITEM_TYPES.EXACT_MATCH">
+                <span>{{ item.expandedInfo1 }}</span>
+                <p class="ma-0">
+                  Consider revising your name unless you can
+                  <v-tooltip top content-class="top-tooltip" transition="fade-transition">
+                    <template v-slot:activator="{ on, attrs }">
+                      <span class="dotted-underline" v-bind="attrs" v-on="on">obtain consent</span>
+                    </template>
+                    <p class="ma-0">
+                      You may be able to use a similar name if you obtain consent or authorization
+                      from the business using the name. If consent is required you will be asked
+                      to submit signed consent to the Business Registry as part of the review process.
+                    </p>
+                  </v-tooltip>
+                  to use this name.
+                </p>
+              </div>
+              <div v-else v-html="item.expandedInfo1">{{ item.expandedInfo1 }}</div>
+              <div v-if="item.expandedInfoBlock1" class="pt-7">
+                <p v-for="line, index in item.expandedInfoBlock1" :key="index" class="ma-0" v-html="line">
+                  {{ line }}
```

Comment:
1 more key instance :) 
Also, is `{{ line }}` required when templating raw html?

Vue ref: https://vuejs.org/v2/guide/syntax.html#Raw-HTML

If not, this would apply to the html templating below.

Reply:
Ah, you're saying every element I have within the raw html needs it's own bind?

Reply:
I think you can do without the templating, ie `{{ line }}` if that property is already contained in your raw html. 
Sorry, i didn't look into the html that you were applying. 

ie `<p v-for="line, index in item.expandedInfoBlock1" :key="index" class="ma-0" v-html="line"></p>` might work.

Reply:
oh cool okay yes I will try that

Reply:
I confirm this is implemented, so I'm resolving this conversation.